### DISCUSSION TITLE
ngnix port change from 80 to 81,  "run" page cannot use

### DISCUSSION THREAD

**Discussion by zhukefucn** [langgenius/dify#6814] [General]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### Content

ngnix port change from 80 to 81, operation page can use correctly，but "run" page cannot use，you must change url ,add :81，then can run .

**Comment by zhukefucn** [comment]
I know how to do .

**Reply by germcell** [reply]
hi，how to resolve this problem？
